tests/models/recurrent_gemma/test_modeling_recurrent_gemma.py

```diff
@@ -0,0 +1,508 @@
+    def is_pipeline_test_to_skip(
+        self, pipeline_test_casse_name, config_class, model_architecture, tokenizer_name, processor_name
+    ):
+        return True
+
+    def setUp(self):
+        # We don't output attentions
+        self.has_attentions = False
+        self.model_tester = RecurrentGemmaModelTester(self)
+        self.config_tester = ConfigTester(self, config_class=RecurrentGemmaConfig, hidden_size=37)
+
+    def test_config(self):
+        self.config_tester.run_common_tests()
+
+    def test_model(self):
+        config_and_inputs = self.model_tester.prepare_config_and_inputs()
+        self.model_tester.create_and_check_model(*config_and_inputs)
+
+    def test_model_various_embeddings(self):
+        config_and_inputs = self.model_tester.prepare_config_and_inputs()
+        for type in ["absolute", "relative_key", "relative_key_query"]:
+            config_and_inputs[0].position_embedding_type = type
+            self.model_tester.create_and_check_model(*config_and_inputs)
+
+    @unittest.skip("Rucurrent gemma does not use legacy cache")
+    @parameterized.expand([(1, False), (1, True), (4, False)])
+    def test_new_cache_format(self, num_beams, do_sample):
+        pass
+
+    def test_save_load_fast_init_from_base(self):
+        pass
+
+    @unittest.skip("RecurrentGemma does not return pkv")
+    def test_past_key_values_format(self):
+        pass
+
+    @unittest.skip("RecurrentGemma only supports sdpa")
+    def test_eager_matches_sdpa_generate(self):
+        pass
+
+    @unittest.skip("RecurrentGemma only supports sdpa")
+    def test_eager_matches_sdpa_inference(self):
+        pass
+
+    @unittest.skip("RecurrentGemma does not return the cache")
+    def test_contrastive_generate_low_memory(self):
+        pass
+
+    @unittest.skip("RecurrentGemma does not return the cache")
+    def test_contrastive_generate_dict_outputs_use_cache(self):
+        pass
+
+    @unittest.skip("RecurrentGemma does not return the cache")
+    def test_contrastive_generate(self):
+        pass
+
+    @unittest.skip("SQRBound is known to have issues with gc")
+    def test_training_gradient_checkpointing_use_reentrant_false(self):
+        pass
+
+    def _check_attentions_for_generate(self, *args, **kwargs):
+        return True  # Model does not return attention
+
+    def test_prompt_lookup_decoding_matches_greedy_search(self):
+        pass
+
+    def test_model_parallelism(self):
+        pass
+
+    def test_model_parallel_beam_search(self):
+        pass
+
+    def _check_past_key_values_for_generate(self, *args, **kwargs):
+        return True
+
+    @unittest.skip("Seems to rely on `past_key_values`")
+    def test_assisted_decoding_matches_greedy_search(self):
+        pass
+
+    @unittest.skip("recurrent gemma's output different if you pad lefr or right. This is expected")
```

Comment:
nit - consistency & spelling

```suggestion
    @unittest.skip("RecurrentGemma's output different if you pad left or right. This is expected")
```